gulp-core-build-karma/karma.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(config) {
 
 
     // list of files / patterns to load in the browser
-      files: [bindPolyfillPath].concat(['lib/tests.js']),
+      files: [bindPolyfillPath].concat(path.join('lib', '**', '.test.js')),
```

Comment:
Also - are you sure you want `lib/**/.test.js` as the glob path? Shouldn't it be `lib/**.test.js`?